<!doctype html><html lang=en-au><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=generator content="Hugo 0.85.0"><title>Popular software - Smyth Lab at WEHI Bioinformatics</title><meta property="og:title" content="Popular software - Smyth Lab at WEHI Bioinformatics"><link href=../favicon.ico rel=icon type=image/x-icon><link rel=stylesheet href=../css/fonts.css media=all><link rel=stylesheet href=../css/main.css media=all></head><body><div class=wrapper><header class=header><nav class=nav><a href=../ class=nav-logo><img src=../images/logo.png width=50 height=50 alt=Logo></a><ul class=nav-links><li><a href=../people/>People</a></li><li><a href=../publications/>Publications</a></li><li><a href=../resources/>Resources</a></li><li><a href=../faqs/>FAQs</a></li></ul></nav></header><main class=content role=main><article class=article><h1 class=article-title>Popular software</h1><div class=article-content><br><h2 id=limma-linear-models-for-microarray-data><code>limma</code>: Linear Models for Microarray Data</h2><p></a><img src=https://bioconductor.org/shields/availability/release/limma.svg></a></span> <img src=https://bioconductor.org/shields/years-in-bioc/limma.svg> <img src=https://bioconductor.org/shields/downloads/release/limma.svg> <img src=https://bioconductor.org/shields//lastcommit/release/bioc/limma.svg></p><br><div class=row><div class=column-software><img src="https://github.com/Bioconductor/BiocStickers/blob/master/limma/limma.png?raw=true" style=width:140px;height:158px;margin-top:-20px></div><p><code>Limma</code> is an R package for the analysis of gene expression microarray data, especially the use of linear models for analysing designed experiments and the assessment of differential expression. <code>Limma</code> provides the ability to analyse comparisons between many RNA targets simultaneously in arbitrary complicated designed experiments. Empirical Bayesian methods are used to provide stable results even when the number of arrays is small. The normalization and data analysis functions are for two-color spotted microarrays. The linear model and differential expression functions apply to all microarray technologies including Affymetrix and other single-channel oligonucleotide platforms. <code>Limma</code> is available on <a href=https://bioconductor.org/packages/release/bioc/html/limma.html>Bioconductor.</p></div><br><h2 id=edger-empirical-analysis-of-digital-gene-expression-data-in-r><code>edgeR</code>: Empirical Analysis of Digital Gene Expression Data in R</h2><p></a><img src=https://bioconductor.org/shields/availability/release/edgeR.svg></a></span> <img src=https://bioconductor.org/shields/years-in-bioc/edgeR.svg> <img src=https://bioconductor.org/shields/downloads/release/edgeR.svg> <img src=https://bioconductor.org/shields//lastcommit/release/bioc/edgeR.svg></p><br><div class=row><div class=column-software><img src="https://github.com/Bioconductor/BiocStickers/blob/7459ae642b36ef1069d7e304dbbcdb76e2acbcee/edgeR/edgeR.png?raw=true" style=width:140px;height:158px></div><p><code>edgeR</code> is an R package for analyzing sequence read count data from genomic sequencing technologies such as RNA-seq, ChIP-seq and ATAC-seq. Like <code>limma</code>, <code>edgeR</code> is particularly designed to detect genes or features that have changed abundance levels between experimental conditions or cell types. Whereas <code>limma</code> is designed to analyze continuous expression values, <code>edgeR</code> is designed to analyze abundance measured in terms of sequence read counts, which enables edgeR to be the underlying analysis engine for a wide range of sequencing technologies. Where <code>limma</code> uses linear models and normality, <code>edgeR</code> uses negative binomial generalized linear models. Like <code>limma</code>, <code>edgeR</code> has the ability to handle arbitrary experimental designs and to borrow information between genes or genomic features. <code>edgeR</code> is available on <a href=https://bioconductor.org/packages/release/bioc/html/edgeR.html>Bioconductor.</a></p></div><p><br><br></p><h1 id=bioconductor-packages>Bioconductor packages</h1><h2 id=limmagui><code>limmaGUI</code></h2><p></a><img src=https://bioconductor.org/shields/availability/release/limmaGUI.svg></a></span> <img src=https://bioconductor.org/shields/years-in-bioc/limmaGUI.svg> <img src=https://bioconductor.org/shields/downloads/release/limmaGUI.svg> <img src=https://bioconductor.org/shields//lastcommit/release/bioc/limmaGUI.svg></p><p>A Graphical User Interface for differential expression analysis of two-color microarray data using the <code>limma</code> package. <code>limmaGUI</code> is available on <a href=https://bioconductor.org/packages/release/bioc/html/limmaGUI.html>Bioconductor.</a></p><h2 id=affylmgui><code>affylmGUI</code></h2><p></a><img src=https://bioconductor.org/shields/availability/release/affylmGUI.svg></a></span> <img src=https://bioconductor.org/shields/years-in-bioc/affylmGUI.svg> <img src=https://bioconductor.org/shields/downloads/release/affylmGUI.svg> <img src=https://bioconductor.org/shields//lastcommit/release/bioc/affylmGUI.svg></p><p><code>affylmGUI</code> provides a menu-driven interface to the <code>affy</code>, <code>gcrma</code>, <code>affyPLM</code> and <code>limma</code> packages for analysing Affymetrix microarray data. The package is developed by James Wettenhall and maintained by Keith Satterley, and available on <a href=https://bioconductor.org/packages/release/bioc/html/affylmGUI.html>Bioconductor.</a></p><h2 id=goseq><code>goseq</code></h2><p></a><img src=https://bioconductor.org/shields/availability/release/goseq.svg></a></span> <img src=https://bioconductor.org/shields/years-in-bioc/goseq.svg> <img src=https://bioconductor.org/shields/downloads/release/goseq.svg> <img src=https://bioconductor.org/shields//lastcommit/release/bioc/goseq.svg></p><p><code>goseq</code> provides functional analysis of gene lists from RNA-Seq experiments, adjusting the gene length bias. The package is authored by Matt Young, and available on <a href=https://bioconductor.org/packages/release/bioc/html/goseq.html>Bioconductor.</a></p><h2 id=csaw><code>csaw</code></h2><p></a><img src=https://bioconductor.org/shields/availability/release/csaw.svg></a></span> <img src=https://bioconductor.org/shields/years-in-bioc/csaw.svg> <img src=https://bioconductor.org/shields/downloads/release/csaw.svg> <img src=https://bioconductor.org/shields//lastcommit/release/bioc/csaw.svg></p><p><code>csaw</code> performs differential binding analyses of ChIP-seq experiments. The package is authored by Aaron Lun. See the Bioconductor Newsletter article on An overview of the <code>csaw</code> package. The package is available on <a href=https://bioconductor.org/packages/release/bioc/html/csaw.html>Bioconductor.</a></p><p><br><br></p><h1 id=cran-r-packages>CRAN R packages</h1><ul><li><p><code>statmod</code> (2001–present) is an R package for biostatistical modelling, including REML analyses, Tweedie generalized linear models, comparative growth curve analysis, and limiting dilution analyses. The package is available on <a href=https://cran.r-project.org/web/packages/statmod/index.html>CRAN.</a></p></li><li><p><code>dglm</code>: Dougle generalized linear models. The package is maintained by Peter Dunn, and available on <a href=https://cran.r-project.org/web/packages/dglm/index.html>CRAN.</a></p></li><li><p><code>tweedie</code>: Functions for Tweedie distributions, exponential family distributions with power-variance functions. The package is maintained by Peter Dunn, and available on <a href=https://cran.r-project.org/web/packages/tweedie/index.html>CRAN.</a></p></li></ul><p><br><br></p><h1 id=web-tools>Web tools</h1><ul><li><p>ELDA: Extreme limiting dilution analysis for comparing stem cell frequencies between populations.</p></li><li><p>Compare Growth Curves. Permutation test for comparing groups of growth curves.</p></li></ul><p><br><br></p></div></article></main><footer class=footer><ul class=footer-links><li><a href=https://github.com/SmythLab class=footer-links-kudos><img src=../images/github.png alt="Link to lab GitHub page"></a></li><li><a href=https://www.wehi.edu.au/people/gordon-smyth class=footer-links-kudos><b>© Smyth Lab<b></a>Bioinformatics Division, Walter and Eliza Hall Institute of Medical Research</li></ul></footer></div></body></html>